app/templates/search/briefs.html

```diff
@@ -82,7 +82,11 @@ <h2>
                 {{ brief.organisation }}
               </li>
               <li>
-                {{ brief.location|join(', ') }}
+                {% for l in brief.location %}
+                {%- if not loop.first and not loop.last %},{% endif %}
```

Comment:
confusing ?

Reply:
is it any clearer with the latest commit?